tests/test-cases/Group7-Regression/7-1-Regression.robot

```diff
@@ -7,4 +7,6 @@ Default Tags  regression
 
 *** Test Cases ***
 Regression test
-    Run Regression Tests
\ No newline at end of file
+    Run Regression Tests
+    	${out}= Run govc vm.info ${vch-name}
+	Should contain ${out} Other 3.x or later Linux (64-bit)
```

Comment:
Spacing is off as well.  Need double spaces between keyword and parms. At least after your Run command and Should Contain